tests/wpunit/Common/Libraries/Uplink/Controller_Test.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace TEC\Common\Libraries\Uplink;
+
+use tad\Codeception\SnapshotAssertions\SnapshotAssertions;
+use TEC\Common\StellarWP\Uplink\Config;
+use TEC\Common\StellarWP\Uplink\Register;
+use TEC\Common\StellarWP\Uplink\Resources\Collection;
+use Tribe__Main;
+use Tribe\Tests\Traits\With_Uopz;
+
+class Controller_Test extends \Codeception\TestCase\WPTestCase {
+	use SnapshotAssertions;
+	use With_Uopz;
+
+	/**
+	 * @var Controller
+	 */
+	private $controller;
+
+	/**
+	 * @before
+	 */
+	public function setup_uplink() {
+		$this->controller = tribe( Controller::class );
+		$this->controller->register_uplink();
+		$this->register_plugin();
+	}
+
+	public function register_plugin() {
+		// Register the test plugin
+		Register::plugin(
+			'common-test-slug',
+			'common-test',
+			'1.0.0',
+			dirname( __FILE__ ),
+			tribe( Tribe__Main::class )
+		);
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_setup_license_fields() {
+		// Retrieve the collection and the specific resource
```

Comment:
The current user is `0` (not logged in), should the license fields render at all?